api/db/thread.js

```diff
@@ -0,0 +1,12 @@
+import mongoose from 'mongoose';
+
+const threadSchema = new mongoose.Schema({
+  name: String,
+  isRead: Boolean,
```

Comment:
can you replace with with readBy: \[\] (I found out scalar isRead on a thread doesn't actually make much sense for a 2+ party room)